_infra/helm/secure-message-v2/values.yaml

```diff
@@ -54,5 +54,9 @@ dns:
 crons:
   markThreadForDeletion:
     name: sm-v2-mark-thread-for-deletion-scheduled-job
-    schedule: "0 1 * * *"
+    schedule: "0 2 * * *"
     target: "batch/mark_thread_for_deletion"
+  deleteThreads:
+    name: sm-v2-delete-threads-scheduled-job
```

Comment:
Is there opportunity to simply name them `secure-message-delete-threads-scheduled-job` and `secure-message-mark-thread-for-deletion-scheduled-job`. Hopefully we can rename the service entirely once v1 is retired so do these need prefixing v2?

Also, are we including the `scheduled-job` in the name because of the existing jobs and are these necessary given the object type provides that info?

Reply:
Just whilst we had both services it was me being explicit as good practise, and yes I did follow the pattern, I am not a huge fan of different names even if the current wouldn't be more choice. However I didn't want to change them

Reply:
I suppose `sm-v2` does just about convey it's dependency (distinct from version 1 of secure-message) although we have avoided abbreviations throughout our services to date.